README.md

```diff
@@ -28,3 +59,33 @@ Example: (Updated)
 More details on experiment-buddy:
 1. experiment-buddy will add tagged commits to a dangling branch 
 2. Supports: Unix based OS
+
+## For the tester
+
+#### Local flow
```

Comment:
what is a flow:?

Reply:
Get a project -> Set git -> set wanddb -> init venv -> Run it -> Deploy it -> [push changes -> update remote -> run experiment/sweep -> ...] -> wait for the results (still WIP)